src/main/java/uk/gov/justice/probation/courtcaseservice/application/QueueConfig.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.probation.courtcaseservice.application;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.justice.hmpps.sqs.HmppsQueueService;
+import uk.gov.justice.hmpps.sqs.HmppsTopic;
+
+import java.util.Optional;
+
+@Configuration
+public class QueueConfig {
```

Comment:
Could this be TopicConfig or some generic class name to use for both the topic and queue instead?